BigBaseV2/src/hooks/script_event_handler.cpp

```diff
@@ -6,7 +6,7 @@ namespace big
 	void format_string(std::string_view player_name, std::string_view protection_type, bool should_log, bool should_notify)
 	{
 		if (should_log)
-			LOG(WARNING) << "BLOCKED_SCRIPT_EVENT";
+			LOG(WARNING) << fmt::format("BLOCKED_SCRIPT_EVENT From: {} Event Type: {}", player_name.data(), protection_type.data());

```

Comment:
Couldn't this be changed to
```cpp
LOG(WARNING) << "BLOCKED_SCRIPT_EVENT From: " << player_name << " Event Type: " << protection_type;
```

Reply:
Should've done this, don't know what I was thinking